tests/rados/test_stretch_site_down.py

```diff
@@ -0,0 +1,266 @@
+
+    # Creating test pool to check the effect of Site down on the Pool IO
+    if not rados_obj.create_pool(pool_name=pool_name):
+        log.error(f"Failed to create pool : {pool_name}")
+        raise Exception("Test execution failed")
+
+    # Collecting the init no of objects on the pool, before site down
+    pool_stat = rados_obj.get_cephdf_stats(pool_name=pool_name)
+    init_objects = pool_stat["stats"]["objects"]
+
+    # Checking which DC to be turned off, It would be either data site or Arbiter site
+    if shutdown_site in [dc_1_name, dc_2_name]:
+        log.debug(f"Proceeding to shutdown one of the data site {dc_1_name}")
+        for host in dc_1_hosts:
+            log.debug(f"Proceeding to shutdown host {host}")
+            if not host_shutdown(gyaml=osp_cred, name=host):
+                log.error(f"Failed to shutdown host : {host}")
+                raise Exception("Test execution Failed")
+
+        log.info(f"Completed shutdown of all the hosts in data site {dc_1_name}.")
+
+        # sleeping for 10 seconds for the DC to be identified as down and proceeding to next checks
+        time.sleep(10)
+
+        # Checking the health status of the cluster and the active alerts for site down
+        # These should be generated on the cluster
+        status_report = rados_obj.run_ceph_command(cmd="ceph report")
+        ceph_health_status = list(status_report["health"]["checks"].keys())
+        expected_health_warns = (
+            "OSD_HOST_DOWN",
+            "OSD_DOWN",
+            "OSD_DATACENTER_DOWN",
+            "MON_DOWN",
+            "DEGRADED_STRETCH_MODE",
+        )
+        if not set(expected_health_warns).issubset(set(ceph_health_status)):
+            log.error(
+                f"We do not have the expected health warnings generated on the cluster.\n"
+                f" Warns on cluster : {ceph_health_status}\n"
+                f"Expected Warnings : {expected_health_warns}\n"
+            )
+
+        log.info(
+            f"The expected health warnings are generated on the cluster. Warnings : {ceph_health_status}"
+        )
+
+        stretch_details = rados_obj.get_stretch_mode_dump()
+        if not stretch_details["degraded_stretch_mode"]:
+            log.error(
+                f"Stretch Cluster is not marked as degraded even though we have DC down : {stretch_details}"
+            )
+            raise Exception("Stretch mode degraded test Failed on the provided cluster")
+
+        log.info(
+            f"Cluster is marked degraded post DC Failure {stretch_details},"
+            f"Proceeding to try writes into cluster"
+        )
+
+    else:
+        log.info("Shutting down arbiter mon site")
+        for host in tiebreaker_hosts:
+            log.debug(f"Proceeding to shutdown host {host}")
+            if not host_shutdown(gyaml=osp_cred, name=host):
+                log.error(f"Failed to shutdown host : {host}")
+                raise Exception("Test execution Failed")
+        log.info(
```

Comment:
Query: Are we not checking the health warning and alerts in case of arbiter site down case? similar to checks added for data site down case.

Reply:
Thanks for point it out. It was missing. I have added the check now.